xendit_pos/models/pos_payment_method.py

```diff
@@ -26,10 +28,35 @@ def _get_payment_terminal_selection(self):
     xendit_pos_test_mode = fields.Boolean(help='Run transactions in the test environment.')
     xendit_pos_latest_response = fields.Char(help='Technical field used to buffer the latest asynchronous notification from Xendit.', copy=False, groups='base.group_erp_manager')
     xendit_pos_latest_diagnosis = fields.Char(help='Technical field used to determine if the terminal is still connected.', copy=False, groups='base.group_erp_manager')
-    
+    xendit_encrypt_key = fields.Char(string="Xendit Encrypt Key", required=True, copy=False)
+
     xendit_invoice_id = ''
     xenditClient = xendit_client.XenditClient
 
+    def __init__(self) -> None:
+        super().__init__()
+
+    @api.onchange('xendit_pos_secret_key')
+    def _onchange_xendit_secret_key(self):
+        if self.xendit_pos_secret_key:
+
+            # Generate the encrypt key using to encrypt secret key
+            if self.xendit_encrypt_key is False or self.xendit_encrypt_key == "":
```

Comment:
random style comments, let's keep to only one style. either `''` or `""`

Reply:
@ikanyu updated.